### ISSUE TITLE
Updating java-test no longer possible

### ISSUE THREAD

**Issue by zenoli** [mason-org/mason-registry#3036]
Unfortunately java-test does not provide `.vsix` files on their release page since v0.39.0 which is what the [current package-specification](https://github.com/mason-org/mason-registry/blob/main/packages/java-test/package.yaml) depends on.

This is probably also why [this](https://github.com/microsoft/vscode-java-test/releases) failed.

A workaround would be to build the package from source, [which seems to be mostly done with npm](https://github.com/microsoft/vscode-java-test/blob/main/.github/workflows/build.yml).

I would be happy to contribute if I could get some pointers on testing the changes to the registry locally.

Is there a way to prevent mason from fetching the registry remotely and somehow point it to a locally pulled registry which I can use for development?

**Comment by s1n7ax** [comment]
Anyone found a solution? `0.39.0` throws this error when looking for tests. Suggestion is to use `0.40.0`
https://github.com/microsoft/vscode-java-test/issues/1625

**Comment by zenoli** [comment]
There is a PR #3083 that build the new libary using npm and maven but @williamboman is still considering other ways to provide pre-build packages instead letting users build it themselves.

[He suggested two workarounds](https://github.com/mason-org/mason-registry/pull/3083#issuecomment-1787355370) to use the fix provided by @brianrodi locally.

**Comment by s1n7ax** [comment]
Hello. I have created a workflow that release build artifacts on new release of java-test. This is daily checking to see if there are new relea are available in the original project that's not available in the current releases. 

NOTE that `0.40.1` release fail to build so at the moment I have used main branch to release the build artifacts.

https://github.com/nvim-java/vscode-java-test-releases

I have a custom mason registry to use the releases from this here

https://github.com/nvim-java/mason-registry

Adding the registry to mason & `Mason` should to the job. `MasonInstall java-test@0.40.1` should install the newest version.

```lua
{
	'williamboman/mason.nvim',
	opts = {
		registries = {
			'github:nvim-java/mason-registry',
			'github:mason-org/mason-registry',
		},
	},
}
```
